packages/gatsby-source-vtex/src/utils.ts

```diff
@@ -36,11 +43,13 @@ export const createDepartmentNode = (
   { id, children, ...department }: Category
 ) => {
   const NODE_TYPE = 'Department'
-  const urlSplited = department.url.split('/')
-  const slug = urlSplited[urlSplited.length - 1]
   const data = {
     ...department,
-    slug,
+    slug: getSlugFromUrl(department.url),
+    subCategories: children.map((subCategory) => ({
+      ...subCategory,
+      slug: getSlugFromUrl(subCategory.url),
```

Comment:
the slug for a subcategory should be `/department/category/subcategory`, why are we adding like `/subcategory` ?

Reply:
ops, good catch. I will fix it!